apps/src/aichat/views/CopyButton.tsx

```diff
@@ -68,21 +69,29 @@ const CopyButton: React.FunctionComponent<{isDisabled: boolean}> = ({
 function chatEventToFormattedString(chatEvent: ChatEvent) {
   const formattedTimestamp = timestampToDateTime(chatEvent.timestamp);
   if (isChatMessage(chatEvent)) {
+    // [{timestamp} - {role}] [FLAGGED AS PROFANITY] | chatMessageText
```

Comment:
would we want to include the whole message with placeholders, so that RTL languages can change the order of the header parts like [{timestamp} - {role}]

Let's re-visit some of these and chat about them. Might be a list of Qs to ask #i18n

Reply:
OK so the question here for i18n I think is - in this string, we have square brackets that are for grouping. Will that be misinterpreted as markdown? Or is it fine to leave those in there?

Reply:
Example of a string with markdown in it: makerSetupAdafruitWindowsDrivers